source/_components/thermostat.heatmiser.markdown

```diff
@@ -14,6 +14,8 @@ ha_category: Thermostat
 
 The `heatmiser` thermostat platform let you control [Heatmiser DT/DT-E/PRT/PRT-E](http://www.heatmisershop.co.uk/heatmiser-slimline-programmable-room-thermostat/) thermostats from Heatmiser.  The module itself is currently setup to work over a RS232 -> RS485 converter, therefore it connects over IP.
 
+Further work would be required to get this setup to connect over Wifi, but the HeatmiserV3 python module being used is a full implementation of the V3 protocol.  If you would like to contribute to making this work over wifi, please contact @andylockran on github.
```

Comment:
I think it would make sense to make it a link to your Github profile/repo

```markdown
[@andylockran on github](https://github.com/andylockran)
```